pkg/lib/ratelimit/error.go

```diff
@@ -2,20 +2,26 @@ package ratelimit
 
 import (
 	"github.com/authgear/authgear-server/pkg/api/apierrors"
+	"github.com/authgear/authgear-server/pkg/api/model"
 )
 
-const rateLimitNameKey = "rate_limit_name"
```

Comment:
Per our discussion, we should keep both `rate_limit_name` and `bucket_name`, and their values should remain unchanged.

This should be temporary until we confirm that our clients aren’t using it in their customizations, right?